asv_bench/benchmarks/benchmarks.py

```diff
@@ -992,4 +993,96 @@ def time_float_32(self, shape, method):
         execute(self.df_ts_float32.fillna(method=method))
 
 
+class TimeLevelAlign:
+    params = [get_benchmark_shapes("TimeLevelAlign")]
+    param_names = ["shape"]
+
+    def setup(self, shape):
+        row1, col1 = shape[0]
+        row2, col2 = shape[1]
+        Nroot = round(math.sqrt(row1))
+        N = Nroot * Nroot
+        self.index = IMPL.MultiIndex(
+            levels=[np.arange(10), np.arange(Nroot), np.arange(Nroot)],
+            codes=[
+                np.arange(10).repeat(N),
+                np.tile(np.arange(Nroot).repeat(Nroot), 10),
+                np.tile(np.tile(np.arange(Nroot), Nroot), 10),
+            ],
+        )
+        self.df = IMPL.DataFrame(
+            np.random.randn(len(self.index), col1), index=self.index
+        )
+        self.df_level = IMPL.DataFrame(np.random.randn(row2, col2))
+        execute(self.df)
+        execute(self.df_level)
+
+    def time_align_level(self, shape):
+        left, right = self.df.align(self.df_level, level=1, copy=False)
+        execute(left)
+        execute(right)
+
+    def time_reindex_level(self, shape):
+        execute(self.df_level.reindex(self.index, level=1))
```

Comment:
This looks like out of scope of `TimeLevelAlign` class.

Reply:
In pandas ASV benchmarks they have added this function in LevelAlign class. What I figured was df.align method and df.reindex by providing index of other dataframe produces the same result. So these were two types of aligning a dataframe. 
Should I remove this method from benchmark or keep it to make it similar to pandas benchmarks?

Reply:
Ok, you can leave it with a comment. Something like: "`reindex` returns the same result as `align`. Approximately the same performance is expected."